src/main/java/frc/robot/subsystems/FlipperSubsystem.java

```diff
@@ -60,12 +90,18 @@ public void flipperScore() {
 
   @Override
   public void periodic() {
+    if (coralDetector1.get() || coralDetector2.get()) {
```

Comment:
🎨 Set these two getters to local variables first, then use them here as well in the log statements. You don't want to `get()` multiple times which could lead to inaccuracies.